PerformanceMetrics/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ set(PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_APPS "${PLUGIN_WEBKITBROWSER_APPS}"
 set(PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_RESIDENT_APP "${PLUGIN_WEBKITBROWSER_RESIDENT_APP}" CACHE BOOL "Enable monitor for the Performance Metrics Resident App plugin")
 set(PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_UX "${PLUGIN_WEBKITBROWSER_UX}" CACHE BOOL "Enable monitor for the Performance Metrics UX plugin")
 set(PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_YOUTUBE "${PLUGIN_WEBKITBROWSER_YOUTUBE}" CACHE BOOL "Enable monitor for the Performance Metrics Youtube plugin")
+set(PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_CLASSNAME "${PLUGIN_WEBKITBROWSER_CLASSNAME}" CACHE BOOL "Enable monitor for the Performance Metrics plugin by classname")
```

Comment:
We do not need to set the default value with another buildflag here (above that was handy to turn on for example monitoring youtube automatically when youtube webkit plugin was enabled).
So I propose to change this to:

set(PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_CLASSNAME OFF CACHE BOOL "Enable monitor for the Performance Metrics plugin by classname")

Now the default is off (not to break the old usage) and PLUGIN_PERFORMANCEMETRICS_WEBKITBROWSER_CLASSNAME  can be set to ON in a build that wants to monitor webkit on classname.

Reply:
Done.